app/Community/RouteServiceProvider.php

```diff
@@ -39,6 +41,15 @@ protected function mapWebRoutes(): void
     {
         Route::middleware(['web', 'csp'])
             ->group(function () {
+                /*
+                 * client-side api calls
+                 */
+                Route::middleware(['auth'])->group(function () {
+                    Route::get('user-game-list', [UserGameListApiController::class, 'index'])->name('api.user-game-list.index');
+                    Route::post('user-game-list/{game}', [UserGameListApiController::class, 'store'])->name('api.user-game-list.store');
+                    Route::delete('user-game-list/{game}', [UserGameListApiController::class, 'destroy'])->name('api.user-game-list.destroy');
```

Comment:
I put all these client-side calls under "api.*" for the name. Figured it probably makes sense to differentiate them from route names that the user can navigate to, etc. Maybe we should adopt this elsewhere in a subsequent change?

Reply:
I'm also wondering if it makes sense for us to start putting these under something like `/request`, eg `/request/user-game-list`.

That may better delineate the URLs from pages that can be navigated to.

Reply:
Tried `api`, but it collides with our existing web API `/API/` prefix, resulting in numerous errors.
Went with `internal-api` in latest.

Reply:
Apologies for the misunderstanding.

I think we should indeed keep "api.*" in the name. However, `prefix` is the route prefix, eg:

`/internal-api/foo`

When a group is given a prefix, all routes are given that prefix in the URL. Hopefully this makes sense.